mlflow/pyfunc/scoring_server/__init__.py

```diff
@@ -170,6 +173,23 @@ def _decode_json_input(json_input):
     )
 
 
+def _split_data_and_params_for_llm_input(json_input, param_schema: ParamSchema):
```

Comment:
Nittiest of nits:

```suggestion
def _split_data_and_params_for_llm_input(json_input, param_schema: Optional[ParamSchema]):
```

(since we do an `if param_schema` check on line 179)